tests/ci_install.sh

```diff
@@ -0,0 +1,43 @@
+set -x   # Show which command is being run
+
+echo "Travis build stage: $TRAVIS_BUILD_STAGE_NAME"
+
+# Step 1: pre-install required packages
+if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+    # Install some dependencies using conda
+    CYTHON=$(grep cython tests/test_prerequirements.txt)
+    NUMPY=$(grep numpy tests/test_prerequirements.txt)
+
+    CARTOPY=$(grep cartopy tests/test_requirements.txt)
+    H5PY=$(grep h5py tests/test_requirements.txt)
+    MATPLOTLIB=$(grep matplotlib tests/test_requirements.txt)
+    SCIPY=$(grep scipy tests/test_requirements.txt)
+    conda install --yes -c conda-forge $CYTHON $NUMPY $CARTOPY $H5PY $MATPLOTLIB $SCIPY
```

Comment:
so won't these packages be installed with both pip and conda on the windows stage, then? the next if/else the windows stage should not be in the MINIMAL==1 stage, right?

Reply:
This is no problem for now because the path `MINIMAL == 1 and TRAVIS_OS_NAME == windows` is not triggered, but you're right that this should be refactored somehow. I'll update.